.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,65 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Describe the bug
+A clear and concise description of what the bug is.
+
+## To Reproduce
+Please provide a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).
+
+>    …Minimal – Use as little code as possible that still produces the same problem
+>   …Complete – Provide all parts someone else needs to reproduce your problem in the question itself
+>   …Reproducible – Test the code you're about to provide to make sure it reproduces the problem
+ 
+
+## Expected behavior
```

Comment:
This is usually paired with an "Actual behavior", did you leave that out intentionally?

Reply:
Isn't that covered by describe the bug/steps to reproduce, I used the templates that github suggests as a basis.